lambdas/eventbrite/handlers/schemas/event-schema.json

```diff
@@ -0,0 +1,260 @@
+{
```

Comment:
Carbon copy of the schema for the farset labs lambda - something to extract in the future.